.github/workflows/ci.yml

```diff
@@ -372,7 +372,7 @@ jobs:
       - name: Submit Dependencies
         uses: scalacenter/sbt-dependency-submission@v2
         with:
-          modules-ignore: rootjs_2.12 rootjs_2.13 rootjs_3 docs_2.12 docs_2.13 docs_3 cats-tests_sjs1_2.12 cats-tests_sjs1_2.13 cats-tests_sjs1_3 rootjvm_2.12 rootjvm_2.13 rootjvm_3 rootnative_2.12 rootnative_2.13 rootnative_3 cats-tests_2.12 cats-tests_2.13 cats-tests_3 bincompattest_2.12 bincompattest_2.13 bincompattest_3 cats-tests_native0.5_2.12 cats-tests_native0.5_2.13 cats-tests_native0.5_3 cats-bench_2.12 cats-bench_2.13 cats-bench_3
+          modules-ignore: cats-root_2.12 cats-root_2.13 cats-root_3 docs_2.12 docs_2.13 docs_3 cats-tests_sjs1_2.12 cats-tests_sjs1_2.13 cats-tests_sjs1_3 cats-root_2.12 cats-root_2.13 cats-root_3 cats-root_2.12 cats-root_2.13 cats-root_3 cats-tests_2.12 cats-tests_2.13 cats-tests_3 bincompattest_2.12 bincompattest_2.13 bincompattest_3 cats-tests_native0.5_2.12 cats-tests_native0.5_2.13 cats-tests_native0.5_3 cats-bench_2.12 cats-bench_2.13 cats-bench_3
```

Comment:
It appears that if the root submodule has a name set through the corresponding SBT settings, we get some duplications when generating `modules-ignore` (checked in [other projects](https://github.com/typelevel/mouse/blob/main/.github/workflows/ci.yml#L328) — it's the same). Interestingly, platform-based postfixes have also disappeared. Not quite sure if we should worry about this though. Tagging a notoriously known expert in the field, cc @armanbilge.

Reply:
I'm not sure if this is the right way to make this change. What happens if you rename `root` to `cats` in `lazy val root = tlCrossRootProject` instead?

Reply:
> I'm not sure if this is the right way to make this change.

It's pretty common in dozens of projects I've seen so far (specifically, adjusting the `name` SBT setting for root-like submodules). IIUC, we can freely give up on the `moduleName` for the root submodule. However, this approach would differ slightly from what we do for the other submodules in the build. That said, it doesn't seem crucial to me.

Aside from that, I wonder why adjusting the moduleName for the root submodule overrides the corresponding names of the platform-based submodules. It should act as a prefix, like `cats-rootJVM`, `cats-rootJS`, etc.